executor/point_get.go

```diff
@@ -44,6 +44,7 @@ func (b *executorBuilder) buildPointGet(p *plannercore.PointGetPlan) Executor {
 		idxVals: p.IndexValues,
 		handle:  p.Handle,
 		startTS: startTS,
+		done:    p.UnsignedHandle && p.Handle < 0,
```

Comment:
What's this?

Reply:
this make `PointGetExecutor`'s Next return no result, we cannot use tabledual,  this just like tabedual, and no need add more flag- -